app/models/search/tag_query.rb

```diff
@@ -23,7 +23,7 @@ def filters
       character_filter,
       suggested_fandom_filter,
       suggested_character_filter,
-      to_wrangle_filter
+      *to_wrangle_filter
```

Comment:
Is the splat needed here, since we already `flatten` the list (and flatten is recursive)?

Reply:
> flatten is recursive

TIL
(my mind likely assumed it wasn't as a default, like [flat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) in JS)